test/filters/oidc/jwks_resolver_test.cc

```diff
@@ -155,6 +155,39 @@ TEST(JwksResolverTest, TestDynamicJwksResolverWithInvalidHttpStatus) {
   EXPECT_EQ(nullptr, resolver.jwks());
 }
 
+TEST(JwksResolverTest,
+     TestDynamicJwksResolver_RequestBy3SecInterval_UntilJwksConfigured) {
```

Comment:
The other test name is CamelCase, we should be consistent and not use underscore.